build.xml

```diff
@@ -712,6 +712,7 @@
 		<property name="spawn" value="true"/>
 		<java jvm="${tomcat.java.jvm}" classname="org.apache.catalina.startup.Bootstrap" fork="true" failonerror="true" spawn="${spawn}" maxmemory="${maxmemory}" if:true="${use.java}">
 			<jvmarg value="-Dcatalina.home=${tomcat.dir}"/>
+			<jvmarg value="--add-opens=java.base/java.lang=ALL-UNNAMED"/>
```

Comment:
Can you add a comment at the end of the line explaining the reason for this line? Just the url to this PR would be good enough as the description of this PR explains it.

Reply:
Done 👍🏻